dipy/viz/actor.py

```diff
@@ -64,46 +68,40 @@ def slicer(data, affine=None, value_range=None, opacity=1.,
     else:
         nb_components = 1
 
-    if value_range is None:
-        vol = np.interp(data, xp=[data.min(), data.max()], fp=[0, 255])
-    else:
-        vol = np.interp(data, xp=[value_range[0], value_range[1]], fp=[0, 255])
-    vol = vol.astype('uint8')
-
+    # Allocate VTK Image Data
+    data = np.copy(data)
```

Comment:
I used vol before so that we don't change the initial data. Or give the impression that we are doing so.